### PR TITLE
Add optimization to template

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

**Review by cschuchardt88 — APPROVED** [review]

**Review by Jim8y — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Fixes # (issue)
 
 <!-- Please delete options that are not relevant. -->
 
+- [ ] Optimization (the change it's only an optimization)
```
Comment (AnnaShaleva) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: s/it's/is

**Comment by vncoelho** [comment]
Optimization and style?

Take a look at https://github.com/neo-project/neo/pull/3248

What would fit there?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Jim8y) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [] Optimization (the change is only an optimization)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ Fixes # (issue)
 
 <!-- Please delete options that are not relevant. -->
 
+- [] Optimization (the change is only an optimization)
```
Comment (Jim8y) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [ ] Optimization (the change is only an optimization)
```

**Comment by Jim8y** [comment]
> Optimization and style?
> 
> Take a look at #3248
> 
> What would fit there?

@vncoelho Optimization maybe,,,,, maybe we can say its an update/upgrade  to dependency/code

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cschuchardt88) [reply .github/PULL_REQUEST_TEMPLATE.md]: Add this one, it's a good one.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ Fixes # (issue)
 
 <!-- Please delete options that are not relevant. -->
 
+- [ ] Optimization (the change is only an optimization)
```
Comment (Jim8y) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [ ] Optimization (the change is only an optimization)
- [ ] Style (the change it's only a code style for better maintenance or standard purpose)
```

Comment (Jim8y) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: @vncoelho Like this?

Comment (vncoelho) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: it's -> is

**Review by Jim8y — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Pull request merged**